examples/Leveraging_model_distillation_to_fine-tune_a_model.ipynb

```diff
@@ -0,0 +1,1055 @@
+   "execution_count": 5,
+   "id": "c3ec2fba-9c99-4cb7-bf56-f13e3816e559",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'\\n    Based on this wine review, guess the grape variety:\\n    This wine is produced by Trimbach in the Alsace region of France.\\n    It was grown in Alsace. It is described as: \"This dry and restrained wine offers spice in profusion. Balanced with acidity and a firm texture, it\\'s very much for food.\".\\n    The wine has been reviewed by Roger Voss and received 87 points.\\n    The price is 24.0.\\n\\n    Here is a list of possible grape varieties to choose from: Gewürztraminer, Pinot Gris, Gamay, Bordeaux-style White Blend, Champagne Blend, Chardonnay, Petit Manseng, Riesling, White Blend, Pinot Blanc, Alsace white blend, Bordeaux-style Red Blend, Malbec, Tannat-Cabernet, Rhône-style Red Blend, Ugni Blanc-Colombard, Savagnin, Pinot Noir, Rosé, Melon, Rhône-style White Blend, Pinot Noir-Gamay, Colombard, Chenin Blanc, Sylvaner, Sauvignon Blanc, Red Blend, Chenin Blanc-Chardonnay, Cabernet Sauvignon, Cabernet Franc, Syrah, Sparkling Blend, Duras, Provence red blend, Tannat, Merlot, Malbec-Merlot, Chardonnay-Viognier, Cabernet Franc-Cabernet Sauvignon, Muscat, Viognier, Picpoul, Altesse, Provence white blend, Mondeuse, Grenache-Syrah, G-S-M, Pinot Meunier, Cabernet-Syrah, Vermentino, Marsanne, Colombard-Sauvignon Blanc, Gros and Petit Manseng, Jacquère, Negrette, Mauzac, Pinot Auxerrois, Grenache, Roussanne, Gros Manseng, Tannat-Merlot, Aligoté, Chasselas, Loin de l\\'Oeil, Malbec-Tannat, Carignan, Colombard-Ugni Blanc, Sémillon, Syrah-Grenache, Sciaccerellu, Auxerrois, Mourvèdre, Tannat-Cabernet Franc, Braucol, Trousseau, Merlot-Cabernet Sauvignon.\\n    \\n    What is the likely grape variety? Answer only with the grape variety name or blend from the list.\\n    '"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def generate_prompt(row, varieties):\n",
+    "    # Format the varieties list as a comma-separated string\n",
+    "    variety_list = ', '.join(varieties)\n",
+    "    \n",
+    "    prompt = f\"\"\"\n",
+    "    Based on this wine review, guess the grape variety:\n",
+    "    This wine is produced by {row['winery']} in the {row['province']} region of {row['country']}.\n",
+    "    It was grown in {row['region_1']}. It is described as: \"{row['description']}\".\n",
+    "    The wine has been reviewed by {row['taster_name']} and received {row['points']} points.\n",
+    "    The price is {row['price']}.\n",
+    "\n",
+    "    Here is a list of possible grape varieties to choose from: {variety_list}.\n",
+    "    \n",
+    "    What is the likely grape variety? Answer only with the grape variety name or blend from the list.\n",
+    "    \"\"\"\n",
+    "    # we could add the following if needed but it looks like it doesn't improve (on 20 rows):\n",
```

Comment:
add the following what? (sorry, a bit confused)

Reply:
removed - old note, ty